plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/topo_diff.py

```diff
@@ -283,6 +290,58 @@ def get_request(host_ip, protocol, resource, headers):
     logging.info("UFM API Request Status: {}, URL: {}".format(response.status_code, request))
     return response
 
+
+def save_cable_validation_cookies(requests_cookiejar, cookie_filename):
+    '''
+    Save cookies to file
+    :param requests_cookiejar:
+    :param cookie_filename:
+    '''
+    with open(cookie_filename, 'wb') as f:
+        pickle.dump(requests_cookiejar, f)
+
+def load_cable_validation_cookies(cookie_filename):
+    '''
+    Load cookie from file
+    :param cookie_filename:
+    '''
+    with open(cookie_filename, 'rb') as f:
```

Comment:
you can get exception if the file is not in the correct format or does not exist

Reply:
call to this function is wrapped with try-except so exception will raise on call level